code/modules/spells/general/ring_of_fire.dm

```diff
@@ -112,12 +122,41 @@
 			sleep(5)
 
 	spawn(duration)
+		if (living_fire)
+			var/list/outter_ring = view(3, get_turf(loc)) - view(3 - 1, get_turf(loc))
+			for (var/turf/T in outter_ring)
+				var/obj/item/projectile/moving_fire/MF = new(T)
+				var/list/possible_targets = list()
+				for (var/mob/living/L in (view(3, MF) - master))
```

Comment:
The reason you're creating MF at all here seems to be so that you can iterate over its view radius with `view(3, MF)`. Can't you do the same thing with just the turf, and then only create the projectile if you have a possible target?

Reply:
That's correct.